collectorApp_2020-03-23.sql

```diff
@@ -0,0 +1,57 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.29)
+# Database: collectorApp
+
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table cartoonChars
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `cartoonChars`;
+
+CREATE TABLE `cartoonChars` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(200) NOT NULL DEFAULT '',
+  `tvshow` varchar(200) NOT NULL DEFAULT '',
+  `species` varchar(200) NOT NULL DEFAULT '',
+  `catchphase` text,
```

Comment:
looks like your missing the image field, did you forget to re-export your database

Reply:
Yep, forgot to re-export the database. I've pushed the new export file up to story-1.